Libplanet/Tx/Transaction.cs

```diff
@@ -460,31 +466,40 @@ public byte[] ToBencodex(bool sign)
         /// <c>false</c>.
         /// </exception>
         [Pure]
-        public IAccountStateDelta EvaluateActions(
+        public IEnumerable<(T, IActionContext, IAccountStateDelta)>
```

Comment:
This triples also seems to be a bit complicated.

Reply:
Fixed: 19089e75db6287a8e9de6eb47b92f5d72eef0d1d.